docs/HOL_Deploying-Using-Puppet/README.md

```diff
@@ -211,24 +137,22 @@ Programs. Users can create their own modules or consume modules from a marketpla
 as the [Forge](http://forge.puppetlabs.com). Some modules on the Forge are official modules that are supported - 
 others are open-source modules uploaded from the community.
 
-There are two major ways to organise Puppet Programs. One is by _site_, and the other is by _environment_. Organizing
-by site allows you to configure a group of nodes from a single catalog. Howeverm, it is better to organize by 
-environment, allowing you to manage different catalogs for different environments such as dev, test and production.
+Puppet Programs are organized by environment, allowing you to manage different catalogs for different environments such as dev, test and production.
```

Comment:
Because Puppet is using modules, classes and manifests for each environment, I would not use the word Programs.